### DISCUSSION TITLE
Why is `removeEditorPanel` deprecated and is there a new alternative?

### DISCUSSION THREAD

**Discussion by binaryfire** [WordPress/gutenberg#61413] [Q&A] [answered]
Hi all

I want to customize the Gutenberg editor for my sites I'm handing over to clients. Eg. remove the 3-dots options dropdown completely, and remove several sections under the "Page" tab in the sidebar. Actually ideally, I'd like to remove the "Page" tab completely.

I've done a lot of searching but haven't found any solutions. Is there a way I can do this? The only thing I've found that does some of this is `removeEditorPanel` but that seems to be deprecated now - does that mean it will stop working at some point?

Thanks

**Comment by ALJ** [comment, marked as answer]
Hi @binaryfire, I can't answer your bigger question of adjusting the Page tab, but as to the `removeEditorPanel` deprecation: that can be found in [PR57012](https://github.com/WordPress/gutenberg/pull/57012). In that PR, `removeEditorPanel` seems to have been added to `core/editor`. Check it out at https://developer.wordpress.org/block-editor/reference-guides/data/data-core-editor/#removeeditorpanel (notice that one is not deprecated) and see if that gives you what you need.

**Reply by binaryfire** [reply]
@ALJ Ah that explains it. Great to see it's still supported. Thanks!
